incubator/node-red/image/config/app-deploy.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: appsody.dev/v1beta1
+kind: AppsodyApplication
+metadata:
+  name: APPSODY_PROJECT_NAME
+spec:
+  # Add fields here
+  version: 1.0.0
+  applicationImage: APPSODY_DOCKER_IMAGE 
+  stack: APPSODY_STACK
+  service:
+    type: NodePort
+    port: APPSODY_PORT
+    annotations:
+      prometheus.io/scrape: 'true'
+  readinessProbe:
```

Comment:
Can you also add the fields for service monitoring to this config file - our Appsody operator now supports creating service monitors CRs if there is a monitoring entry. See for [nodejs-express](https://github.com/appsody/stacks/blob/master/incubator/nodejs-express/image/config/app-deploy.yaml)